bin/auth-api/prisma/migrations/migration_lock.toml

```diff
@@ -1,3 +1,3 @@
 # Please do not edit this file manually
-# It should be added in your version-control system (e.g., Git)
+# It should be added in your version-control system (i.e. Git)
```

Comment:
Although the comma should indeed be removed, e.g. is the right term here since we're listing  examples. i.e. is used for clarifications

This is a side note though, no real need to change it 😄